### DISCUSSION TITLE
Olarm API not returning JSON.

### DISCUSSION THREAD

**Discussion by rainepretorius** [rainepretorius/olarm-ha-integration#50] [Announcements]
There is a current issue with the Olarm API where some users (especially the ones with two or more devices or many sensors(32+)) are experiencing an error of "Too Many Requests". For the time being until Olarm has fixed this from their side, the refresh interval should be greater than 30s. I am awaiting feedback from Olarm to see how we can work around this issue.

Kind Regards,
Raine

**Comment by rwoeke** [comment]
Thank you @rainepretorius . At least you now know its an Olarm issue. I deleted the integration and re-added it with 31seconds. Lets see if that helps. Is there a config where I can force to only use one device? It's not saving when using the checkboxes on the UI.

**Reply by rainepretorius** [reply]
> Thank you @rainepretorius . At least you now know its an Olarm issue. I deleted the integration and re-added it with 31seconds. Lets see if that helps. Is there a config where I can force to only use one device? It's not saving when using the checkboxes on the UI.

Thank you for reporting the bug that it does not enable only one device but all. I will implement the fix asap.
